### PR TITLE
Posting to Teams workflow should also update PRs

### LINKED ISSUE #6367: Make sure Webhook still works in order to post new issues/PRs from Github to Teams

### PR DESCRIPTION
**Pull request by dmonroym**
Currently our workflow is only posting new issues in our teams channel. Since we moved to a secret we believe this is giving us problems with PRs.
This PR is a placeholder for now and it's also helping us test the changes (hence the closing and reopening of it)

Current status: Reached out to Github support for help

Fixes #6367

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/project-system/pull/6372)

### CODE REVIEW

File: .github/workflows/teams-notifications.yml [hunk 1]
```diff
@@ -12,8 +12,6 @@ jobs:
     
     steps:
       - name: Notify
-        uses: davidwengier/PostAdaptiveCard@v1.0.1
+        uses: davidwengier/PostAdaptiveCard@v1.0.2
```
Comment (ocallesp) [inline .github/workflows/teams-notifications.yml:15]: What is the purpose of 'uses' in a yml ? and why there is a specific username ?

Comment (dmonroym) [reply .github/workflows/teams-notifications.yml:15]: It's not a specific username but a reference to a repo. We're using the Github action that David created and it's hosted in https://github.com/davidwengier/PostAdaptiveCard

**Pull request merged**